downstream/modules/platform/ref-general-inventory-variables.adoc

```diff
@@ -35,24 +35,56 @@ Enter your Red Hat Registry Service Account credentials in `registry_username` a
 When `registry_url` is `registry.redhat.io`, username and password are required if not using a bundle installer.
 
 For more information, see xref:proc-set-registry-username-password[Setting registry_username and registry_password].
-| *`registry_url`* | Default = `registry.redhat.io`.
-| *`registry_username`* | `registry_username` is only required if a non-bundle installer is used.
+| *`registry_url`* |`registry_url` | URL for the registry source. 
+
+Default = `registry.redhat.io`.
+| *`registry_username`* |`registry_username` |`registry_username` is only required if a non-bundle installer is used.
```

Comment:
Same as above 

""The registry_password is required if a non-bundle installer is used"